packages/http-client-java/package.json

```diff
@@ -25,7 +25,7 @@
     "node": ">=18.0.0"
   },
   "scripts": {
-    "clean": "rimraf ./dist ./temp ./generator/target/emitter.jar",
+    "clean": "rimraf ./dist ./temp ./emitter/temp ./generator/target/emitter.jar",
```

Comment:
Need to delete `emitter/temp`, else that folder would prevent `npm run build` from compile the whole module.